webnode/src/redux/epics/treasure-hunt-epics.js

```diff
@@ -18,15 +18,15 @@ const performPowEpic = (action$, store) => {
     .ofType(treasureHuntActions.TREASURE_HUNT_PERFORM_POW)
     .mergeMap(action => {
       const { treasureHunt } = store.getState();
-      const { dataMapHash, treasure, chunkIdx, numberOfChunks } = treasureHunt;
+      const { dataMapHash, treasure, chunkIdx } = treasureHunt;
 
       const address = iota.toAddress(
         iota.utils.toTrytes(Datamap.obfuscate(dataMapHash))
       );
       // const address =
       //   "HT9MZQXKVBVT9AYVTISCLELYWXTILJDIMHFQRGS9YIJUIRSSNRZFIZCHYHQHKZIPGYYCSUSARFNSXD9UY";
 
-      const [_obfHash, nextDataMapHash] = Datamap.hashChain(dataMapHash);
+      const [_obfHash, nextDataMapHash] = Datamap.hashChain(dataMapHash); //eslint-disable-line
```

Comment:
is `//eslint-disable-line` comment to tell the linter not to lint this line?

Reply:
yes, it logs that its not being used. That array assignment wont get screwed up if i remove the variable, will it?

Reply:
i think it will get messed up, i guess we have to leave the comment in